app/models/slate_lineup.py

```diff
@@ -32,12 +34,21 @@ async def get_slate_lineup(slate_lineup_id: str, user_id: Optional[str] = None,
         :param slate_count: int, 0 = no slates, > 0 include this many slates
         :return: SlateLineupModel object
         """
-        experiment = SlateLineupConfigModel.get_experiment_from_slate_lineup(slate_lineup_id)
-        slates = await SlateLineupModel.__get_slates_from_experiment(slate_lineup_id,
-                                                                     experiment,
-                                                                     user_id=user_id,
-                                                                     recommendation_count=recommendation_count,
-                                                                     slate_count=slate_count)
+        try:
+            experiment, slates = await SlateLineupModel.__get_slate_lineup_without_fallback(
+                slate_lineup_id, user_id, recommendation_count, slate_count
+            )
+
+        except PersonalizationError as e:
+            slate_lineup_id = app.config.personalization_fallback_slate_lineup.get(slate_lineup_id,None)
+            if slate_lineup_id:
+                # Retry getting a lineup, but this time with the fallback slate_lineup_id
+                experiment, slates = await SlateLineupModel.__get_slate_lineup_without_fallback(
+                    slate_lineup_id, user_id, recommendation_count, slate_count
```

Comment:
Let's remove `without_fallback` from the name of this function since we also use it to fetch the fallback.

Reply:
Good point, I can now see how that's confusing. Let's rename `get_slate_lineup` to `get_slate_linup_with_fallback` then.